include/conf/find.hpp

```diff
@@ -10,6 +10,17 @@ namespace color_coded
   {
     namespace fs = boost::filesystem;
 
+    inline fs::directory_iterator find_file(fs::path const &dir, std::string const &file)
+    {
+      fs::directory_iterator const end;
```

Comment:
I should've been more clear, sorry. We could make `end` static, since we create it once and it never changes. All the functions are inline since they're in headers; we have no idea which of them actually get inlined. :)